Sources/Internal/LayoutItem.swift

```diff
@@ -66,13 +70,26 @@ extension LayoutItem.ItemType: Comparable {
       return lhsMonth < rhsMonth
     case let (.dayOfWeekInMonth(_, lhsMonth), .day(rhsDay)):
       return lhsMonth <= rhsDay.month
+    case let (.dayOfWeekInMonth(_, lhsMonth), .monthFooter(rhsMonth)):
+      return lhsMonth < rhsMonth
 
     case let (.day(lhsDay), .day(rhsDay)):
       return lhsDay < rhsDay
     case let (.day(lhsDay), .monthHeader(rhsMonth)):
       return lhsDay.month < rhsMonth
     case let (.day(lhsDay), .dayOfWeekInMonth(_, rhsMonth)):
       return lhsDay.month < rhsMonth
+    case let (.day(lhsDay), .monthFooter(rhsMonth)):
+      return lhsDay.month < rhsMonth
```

Comment:
```suggestion
      return lhsDay.month <= rhsMonth
```

Same reasoning here! Even in the same month, every single day item (1-31) will be "before" the month footer at the bottom.